typings/index.d.ts

```diff
@@ -0,0 +1,53 @@
+declare module 'out-of-band-cache' {
+  type JSONSerializable = Object | string | number | boolean | Date;
+
+  type GetResult = (value: JSONSerializable) => boolean;
```

Comment:
I'd inline or rename this type; the name is confusing since this isn't the result of a cache get. Perhaps `ShouldCacheCheck`?

Reply:
Ah this is mis-understanding on my part, this should have been an interface looking at [lib/index.js:L9-13](https://github.com/godaddy/out-of-band-cache/blob/master/lib/index.js#L10)

```suggestion
/**
 * @typedef GetResult
 * @prop {JSONSerializable} value The value of the cache item
 * @prop {Boolean} fromCache Whether the value was pulled from the cache or refreshed from the source API
 */
```

thanks for the input 👍